HOW_TO_RELEASE.md

```diff
@@ -57,47 +57,26 @@ upstream        https://github.com/pydata/xarray (push)
       pytest
       ```
  9. Check that the ReadTheDocs build is passing.
-10. Tag the release:
-      ```sh
-      git tag -a v{0.X.Y} -m 'v{0.X.Y}'
-      ```
-11. Ensure the dependencies for building are installed:
-      ```sh
-      pip install setuptools-scm twine wheel
-      ```
-12. Build source and binary wheels for PyPI:
-      ```sh
-      git clean -xdf  # This removes any untracked files!
-      git restore -SW .  # This removes any tracked changes!
-      python setup.py bdist_wheel sdist
-      ```
-13. Use twine to check the package build:
-      ```sh
-      twine check dist/xarray-{0.X.Y}*
-      ```
-14. Use twine to register and upload the release on PyPI. Be careful, you can't
-    take this back!
-      ```sh
-      twine upload dist/xarray-{0.X.Y}*
-      ```
-    You will need to be listed as a package owner at
-    <https://pypi.python.org/pypi/xarray> for this to work.
-15. Push your changes to master:
-      ```sh
-      git push upstream master
-      git push upstream --tags
-      ```
-16. Update the stable branch (used by ReadTheDocs) and switch back to master:
+10. Issue the release on GitHub. Click on "Draft a new release" at
+    <https://github.com/pydata/xarray/releases>. Type in the version number (with a "v")
+    and paste the release summary in the notes.
+11. This should automatically trigger an upload of the new build to PyPI via GitHub Actions.
+    Check this has run [here](https://github.com/pydata/xarray/actions/workflows/pypi-release.yaml),
+    and that the version number you expect is displayed [on PyPI](https://pypi.org/project/xarray/)
+12. Update the stable branch (used by ReadTheDocs) and switch back to master:
      ```sh
       git switch stable
       git rebase master
       git push --force upstream stable
       git switch master
      ```
+    You may need to first fetch it with `git fetch upstream`,
+    and check out a local version with `git checkout -b stable upstream/stable`.
+
     It's OK to force push to `stable` if necessary. (We also update the stable
     branch with `git cherry-pick` for documentation only fixes that apply the
     current released version.)
-17. Add a section for the next release {0.X.Y+1} to doc/whats-new.rst:
+13. Add a section for the next release {0.X.Y+1} to doc/whats-new.rst:
      ```rst
      .. _whats-new.{0.X.Y+1}:
```

Comment:
the braces are often not removed, so it might be better to remove them (also for the heading, but I can't seem to extend the suggestion)
```suggestion
  .. _whats-new.0.X.Y+1:
```